.pre-commit-config.yaml

```diff
@@ -101,7 +101,7 @@ repos:
       - id: pyupgrade
         args: ["--keep-percent-format"]
   - repo: https://github.com/PyCQA/isort
-    rev: 5.5.1
+    rev: 5.12.0
```

Comment:
For this, wait until https://github.com/OCA/l10n-netherlands/pull/385 is merged and then rebase.